From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ecmel=20Berk=20Canl=C4=B1er?= <me@ecmelberk.com>
Date: Tue, 28 Dec 2021 19:25:30 +0300
Subject: Add warning about being an unofficial patch

This is just to help mod developers and other people who don't need to
care about this patchset to ignore any issues that are not their fault.
---
 src/init.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/init.c b/src/init.c
index 52ade583..1d7ca5d5 100644
--- a/src/init.c
+++ b/src/init.c
@@ -249,6 +249,12 @@ GLFWAPI int glfwInit(void)
     _glfw.timer.offset = _glfwPlatformGetTimerValue();
 
     glfwDefaultWindowHints();
+
+    fprintf(stderr, "!!! Patched GLFW from https://github.com/Admicos/minecraft-wayland\n"
+         "!!! If any issues with the window, or some issues with rendering, occur, "
+         "first try with the built-in GLFW, and if that solves the issue, report there first.\n"
+         "!!! Use outside Minecraft is untested, and things might break.\n");
+
     return GLFW_TRUE;
 }
 
